octopoes/octopoes/repositories/ooi_repository.py

```diff
@@ -68,6 +69,19 @@ def __init__(self, event_manager: EventManager):
     def get(self, reference: Reference, valid_time: datetime) -> OOI:
         raise NotImplementedError
 
+    def get_history(
+        self,
+        reference: Reference,
+        *,
+        sort_order: str = "asc",  # Or: "desc"
```

Comment:
If you use a union of literals as the type hint, you get some static type checking for free

Reply:
Importing `typing.Literal` gave me rather strange bugs that seemed a bit "perhaps-not-worth-the-effort-now":
- https://github.com/minvws/nl-kat-coordination/actions/runs/7048457913/job/19184659208
- https://github.com/minvws/nl-kat-coordination/actions/runs/7048458437/job/19184660541

Reply:
Yes, that's because you're importing `Literal` from `pyparsing`, it should be imported from `typing` obviously

Reply:
That's in a different file where we do use the `pyparsing.Literal` though :thinking:

Reply:
https://github.com/minvws/nl-kat-coordination/issues/2093